### PR TITLE
Fix broken CI

### PR DESCRIPTION
**Pull request by amitsagtani97**
Fix the broken CI

### CODE REVIEW

**Comment by amitsagtani97** [comment]
@julialongtin @supersven The last green CI was found to be working with v1.43.0 of hcloud module(https://registry.terraform.io/providers/hetznercloud/hcloud/latest)
the most recent version 1.44 onwards have a lot of changes which are not supported by many existing modules in our code.

Also, I tried to play around with different versions of terraform(0.14 and 1.0),  but it didn't worked. 
Looks like this is the good fix at the current set of dependencies(both internaland external like kubespray)

File: offline/cd.sh [hunk 1]
```diff
@@ -4,12 +4,11 @@ set -euo pipefail
 
 function cleanup {
   (cd terraform/examples/wire-server-deploy-offline-hetzner ; terraform destroy -auto-approve)
-  echo done
 }
 trap cleanup EXIT
-
 (cd terraform/examples/wire-server-deploy-offline-hetzner ; terraform init ; terraform apply -auto-approve )
 adminhost=$(cd terraform/examples/wire-server-deploy-offline-hetzner ; terraform output adminhost)
+adminhost="${adminhost//\"/}"
```
Comment (supersven) [inline offline/cd.sh:11]: Why is this needed? Has the output format of `terraform output` changed between versions (`0.13` -> `1.x`)?

**Review by supersven — CHANGES_REQUESTED** [review]
Thanks a lot for investigating in this!

- I think it would be good to cleanup the commit history (squash / drop / ...)
- We should have an answer for this: https://github.com/wireapp/wire-server-deploy/pull/666/files#r1357993681
- Are you sure that all Terraform usages still work? Or, does this PR need additional manual testing?

**Comment by amitsagtani97** [comment]
> Thanks a lot for investigating in this!
> 
> * I think it would be good to cleanup the commit history (squash / drop / ...)
> * We should have an answer for this: https://github.com/wireapp/wire-server-deploy/pull/666/files#r1357993681
> * Are you sure that all Terraform usages still work? Or, does this PR need additional manual testing?



> Thanks a lot for investigating in this!
> 
> * I think it would be good to cleanup the commit history (squash / drop / ...)
> * We should have an answer for this: https://github.com/wireapp/wire-server-deploy/pull/666/files#r1357993681
> * Are you sure that all Terraform usages still work? Or, does this PR need additional manual testing?

 - Fixed the history.
 - There were extra double quotes returned from the terraform output from the new version, so had to remove those.
 - As of now, I tried to run "terraform init" command in all the environment directory in the wire-server-deploy and it returned no warnings or error message.

**Pull request merged**